Cargo.toml

```diff
@@ -13,7 +13,7 @@ incremental = false
 overflow-checks = true
 
 [workspace.dependencies]
-neutron-sdk = { git = "https://github.com/neutron-org/neutron-sdk", branch = "main" }
+neutron-sdk = { package = "neutron-sdk", git = "https://github.com/neutron-org/neutron-sdk", branch = "feat/proposal-votes" }
```

Comment:
change this line after appropriate [PR](https://github.com/neutron-org/neutron-sdk/pull/129) will be merged and published.